### ISSUE TITLE
Vimeo: 'qualities' is not a function

### ISSUE THREAD

**Issue by wplit** [vidstack/player#1024]
When building with the latest v1.7.0 version, there's an error shows in the vidstack-video-TX2o91bs.js file if I try any vimeo URL.

![SCR-20231130-ohfx](https://github.com/vidstack/player/assets/43051571/ec12aa7e-33a3-437a-b948-bf5c7f586aeb)

I looked at the original source file, and the part it's referring to is the qualities[...]

<img width="608" alt="SCR-20231130-opuz" src="https://github.com/vidstack/player/assets/43051571/e0e130d1-5abb-4c99-9576-20a85447c95d">

Unable to recreate on the stackblitz. But I started from scratch a couple of times and made sure I had the latest node modules, so I don't think it's something I'm doing wrong. But strange how I can't replicate on there, it works fine.

(maybe just due to  the modification of the code, i notice this doesn't happen on the stackblitz version of the files)

**Comment by mihar-22** [comment]
How are you building and serving it? If you can provide a repro or some instructions that'd help me find out what's up.

**Comment by wplit** [comment]
Turns out I that when I was running the 'npm run build', this time round (unlike when doing it with previous versions) the JS files being split out included a prod...js file that needed to be included in the HTML, not the index...js. file. 

Not sure why this is, or if it's because of something I did wrong, but anyway including the correct file was all that was needed to fix.

I'll keep this issue open as @qlaffont seems to have run into something vimeo related, will close if another issue is created for that.
